app/controllers/api/v1/search_controller.rb

```diff
@@ -368,6 +368,15 @@ def index
           end
         end
 
+        # Pin public DE pilot study to second row of home page default study list
+        # TODO (SCP-4374): Remove this block once frontend uses is_differential_expression_enabled
+        if Rails.env.production? && !params[:terms].present? && !@facets.any?
```

Comment:
Looks good, just some linting suggestions:
```suggestion
        if Rails.env.production? && params[:terms].blank? && @facets.empty?
          de_accession = "SCP1671"
          differential_expression_study = @studies.find { |study| study.accession == de_accession }
          @studies.delete_if { |study| study.accession == de_accession }
          @studies.insert(1, differential_expression_study)
        end
```

Reply:
Are these linting suggestions you get in RubyMine, @bistline?  If so, I can tinker with my VS Code Ruby linter to hopefully iron out such wrinkles before my next review.  (I pulled this syntax from close above.)